src/course-page-resources/course-page/CoursePageConfigCard.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCog } from '@fortawesome/free-solid-svg-icons';
+import { Button } from '@edx/paragon';
+import classNames from 'classnames';
+
+import messages from '../messages';
+
+function CoursePageConfigCard({
+  intl,
+  id,
+  title,
+  description,
+  pageStatus,
+  showSettings,
+}) {
+  const cardSettings = 'bg-white shadow p-3 m-0 my-3 course-page-title';
+  const cardClasses = classNames('card', cardSettings, { disabled: !pageStatus });
+  const cardSubSettings = 'bg-white px-3 py-0';
+
+  const buttonSettings = 'd-flex mt-0 justify-content-center align-items-center';
+  const buttonClasses = classNames(buttonSettings, { visible: !pageStatus, invisible: pageStatus });
```

Comment:
```suggestion
  const buttonClasses = classNames('d-flex mt-0 justify-content-center align-items-center', { visible: !pageStatus, invisible: pageStatus });
```
Don't make it set visibility, just don't add it to the dom if it shouldn't show up unless there is a good reason to have it invisible (for screenreaders for instance). 
Also buttonClasses is a bit odd since they don't apply to a button.

Reply:
This has been cleaned up entirely.